power_grid_model_c/power_grid_model_cpp/include/power_grid_model_cpp/basics.hpp

```diff
@@ -0,0 +1,35 @@
+// SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#pragma once
+#ifndef POWER_GRID_MODEL_CPP_BASICS_HPP
+#define POWER_GRID_MODEL_CPP_BASICS_HPP
+
+#ifdef PGM_DLL_EXPORTS
+#error "Cannot export dynamic targets from the PGM C API wrapper. Please disable PGM_DLL_EXPORT."
+#endif
+
+#include <exception>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "power_grid_model_c/basics.h"
```

Comment:
here and in other files: we organize the include order to current lib headers, then other custom lib headers, then 3rd party, then STL headers

```suggestion
#include "power_grid_model_c/basics.h"

#include <exception>
#include <memory>
#include <string>
#include <vector>
```

Reply:
Got it. Will do.